vip-helpers/vip-clean-term-cache.php

```diff
@@ -52,8 +54,17 @@ public function edited_term_taxonomy_action( $tt_id, $taxonomy ) {
 		// phpcs:ignore WordPress.DB.DirectDatabaseQuery
 		$objects = $wpdb->get_var( $wpdb->prepare( "SELECT COUNT(*) FROM $wpdb->term_relationships WHERE term_taxonomy_id = %d", $tt_id ) );
 
+		$num_objects = intval( $objects );
+
+		// Allows admin users to see saved changes in the edit terms page, when editing a topic
+		// Short circuits scheduling and runs the method synchronously
+		if ( ( $num_objects < $this->admin_limit ) && is_admin() ) {
```

Comment:
I don't think `is_admin` is what we want here because it fires on AJAX requests too? cc @WPprodigy

Reply:
Maybe `wp_doing_ajax()`? https://developer.wordpress.org/reference/functions/wp_doing_ajax/

Reply:
I expanded the condition to include a negative check for wp_doing_ajax. What do you think about this?

```PHP
    if ( ( $num_objects < $this->admin_limit ) && is_admin() && ! wp_doing_ajax() ) {
```